packages/webapp/src/pages/Logs/ShowOperation.tsx

```diff
@@ -0,0 +1,133 @@
+import { useMemo } from 'react';
+import Info from '../../components/ui/Info';
+import { useGetOperation } from '../../hooks/useLogs';
+import { useStore } from '../../store';
+import { OperationTag } from './components/OperationTag';
+import { StatusTag } from './components/StatusTag';
+import { elapsedTime, formatDateToLogFormat } from '../../utils/utils';
+import { Link } from 'react-router-dom';
+import { CalendarIcon, ClockIcon, ExternalLinkIcon } from '@radix-ui/react-icons';
+import { SearchInOperation } from './components/SearchInOperation';
+import { Skeleton } from '../../components/ui/Skeleton';
+
+export const ShowOperation: React.FC<{ operationId: string }> = ({ operationId }) => {
+    const env = useStore((state) => state.env);
+    const { operation, loading, error } = useGetOperation(env, { operationId });
+
+    const duration = useMemo<string>(() => {
+        if (!operation) {
+            return '';
+        }
+        if (!operation.endedAt || !operation.startedAt) {
+            return 'n/a';
+        }
+
+        return elapsedTime(new Date(operation.startedAt), new Date(operation.endedAt));
+    }, [operation]);
+    const createdAt = useMemo(() => {
+        return operation?.createdAt ? formatDateToLogFormat(operation?.createdAt) : 'n/a';
+    }, [operation?.createdAt]);
+
+    if (loading) {
+        return (
+            <div className="py-6 px-6 flex flex-col gap-9">
+                <h3 className="text-xl font-semibold text-white flex gap-4 items-center">Operation Details</h3>
+                <Skeleton className="h-4 w-[250px]" />
```

Comment:
If the `Skeleton` spacing needs to change seems it'll be annoying to change it in all these different places

Reply:
by spacing you mean height?

Reply:
yeah, height and weight. My comment should have said:

If the `Skeleton` element needs to change

Reply:
Skeleton is made to be flexible to reflect the final UI so it's bound to be modified where it needs to. I'll set the default height which is more common but the width is context specific.

For the ref, this is that: https://ui.shadcn.com/docs/components/skeleton